scripts/run-webservice-script.sh

```diff
@@ -4,8 +4,8 @@ set -o pipefail
 set -o nounset
 set -o xtrace
 
-#JAR_PATH="https://artifacts.oicr.on.ca/artifactory/collab-release/io/dockstore/dockstore-webservice/${npm_package_config_webservice_version}/dockstore-webservice-${npm_package_config_webservice_version}.jar"
-JAR_PATH="https://3855-33383826-gh.circle-artifacts.com/0/tmp/artifacts/dockstore-webservice-1.8.6-SNAPSHOT.jar"
+JAR_PATH="https://artifacts.oicr.on.ca/artifactory/collab-release/io/dockstore/dockstore-webservice/${npm_package_config_webservice_version}/dockstore-webservice-${npm_package_config_webservice_version}.jar"
+#JAR_PATH="https://3855-33383826-gh.circle-artifacts.com/0/tmp/artifacts/dockstore-webservice-1.8.6-SNAPSHOT.jar"
```

Comment:
Still needs code comment and link to wiki
 (imagine we're onboarding someone in two weeks and they need to make/test a release)